examples/tars-http-server/src/controller/IndexController.php

```diff
@@ -85,21 +85,7 @@ public function actionFile()
         $this->sendRaw('success:'.var_export($this->request->data['files']['image'], true));
     }
 
-    // curl "172.16.0.161:28887/Index/testSelf?a=b" -i
```

Comment:
I thought this might cause request circling. I tried this case, nothing responds. I might have not fully understood this case. The reason why I decided to remove this case is to make sure no fresh learners get confused.